packages/react-components/react-migration-v8-v9/src/components/utils.ts

```diff
@@ -0,0 +1,10 @@
+export function getHTMLAttributes<T extends object>(props: T, allV8PropsSet: Set<string>): T {
```

Comment:
Consider putting this in shimCheckboxProps.ts as it is only called from there. If it is useful to other shims we could move it higher in the source tree later.

Reply:
Yes, it's a common function shared in other shims.